config/config-api/src/com/thoughtworks/go/domain/packagerepository/PackageRepositories.java

```diff
@@ -111,5 +111,16 @@ private void validateFingerprintUniqueness() {
             }
         }
     }
+
+    public PackageDefinition findPackageWith(String packageId) {
```

Comment:
Can we keep name and behavior consistent with the other method
```
findPackageRepositoryHaving
```

Maybe findPackageDefinitionWith
Also how the return us handled 
```
        throw new RuntimeException(format("Could not find repository for given package id:[%s]", packageId));
```